docs/internals/contributing/writing-code/submitting-patches.txt

```diff
@@ -262,16 +268,20 @@ documentation.
 Contribution checklist
 ======================
 
-Use this checklist to review a pull request. If you are reviewing a pull
-request that is not your own and it passes all the criteria below, please set
-the "Triage Stage" on the corresponding Trac ticket to "Ready for checkin".
+Use this checklist to review to review a pull request.
+If you are reviewing a :ref:`non-trivial-contribution`,
+first ensure it has an accepted ticket.
+If the pull request passes all the criteria below and is not your own,
+please set the "Triage Stage" on the corresponding Trac ticket to "Ready for checkin".
 If you've left comments for improvement on the pull request, please tick the
 appropriate flags on the Trac ticket based on the results of your review:
 "Patch needs improvement", "Needs documentation", and/or "Needs tests". As time
 and interest permits, mergers do final reviews of "Ready for checkin" tickets
 and will either commit the changes or bump it back to "Accepted" if further
-work needs to be done. If you're looking to become a merger, doing thorough
-reviews of contributions is a great way to earn trust.
+work needs to be done.
+
+If you're looking to become a member of the Review and Triage team, doing thorough reviews
+of contributions is a great way to earn trust.
```

Comment:
Thank you for doing this, I think let's pull this out into a separate commit (or separate PR) as it more of a "side-quest" than what we're trying to achieve here 👍

Reply:
Thank you, this also needs wrapping at 80 characters